src/strings/en-gb.json

```diff
@@ -401,6 +401,7 @@
     "ThisWizardWillGuideYou": "This wizard will help guide you through the setup process. To begin, please select your preferred language.",
     "TheseSettingsAffectSubtitlesOnThisDevice": "These settings affect subtitles on this device",
     "TabServer": "Server",
+    "TabStudios": "Movie Studios",
```

Comment:
Only changes to the `en-us` file should be made in PRs. Other languages need to be added via translate.jellyfin.org after merging.